_docs/transaction-flow.md

```diff
@@ -1188,6 +1188,45 @@ The state of the recipient. Can be one of the following:
 
 If there is a specific error for the recipient, you can find a description of the error here.
 
+### state_reason_details
+
+If there is a specific error for the recipient, you can find details here.
+It contains following fields:
+
+#### code
+
+Status code of failed transaction.
+
+There are six group of errors based on error codes:
+* 0 - transaction is paid
+* 1x - transaction is awaiting
+* 2x - recipient action required
+* 3xx - temporary error. We will retry the transaction at a later date.
+* 4xx - recipient error. Update recipient details or cancel this transaction.
+* 5xx - sender / fatal error hence transaction cannot be processed
+
+#### category
+
+Main category of error.
+
+List of categories:
```

Comment:
Let's add more details on what each of these mean. We can potentially merge this section with the one above